Add overview for order drop-in (#144)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/order/index.mdx

---

> title: Order initialization

Reviewer: ```suggestion
title: Order overview
```

description: Learn about the features and functions of the order drop-in component.
---

import { Badge } from '@astrojs/starlight/components';
import OptionsTable from '@components/OptionsTable.astro';
import Diagram from '@components/Diagram.astro';

> The Order drop-in component provides a comprehensive set of tools and containers designed to manage and display order-related data across various pages and scenarios. It simplifies the implementation of order management functionality and supports seamless integration with both customer accounts and guest user workflows.

Reviewer: ```suggestion
The order drop-in component provides a comprehensive set of tools and containers designed to manage and display order-related data across various pages and scenarios. It simplifies the implementation of order management functionality and supports seamless integration with both customer accounts and guest user workflows.
```

## Architecture

> The Order drop-in consists of multiple containers that display order details on different pages, such as:

Reviewer: ```suggestion
The order drop-in component consists of multiple containers that display order details on different pages, such as:
```

- Order data containers display order details within the customer account, guest user areas, and on the order confirmation page.
- Returned merchandise authorization (RMA) containers guide users through the return process and display a list of created return requests.

> - The `OrderSearch` container enables guest users to locate their orders using a combination of email, zip code, and order number. This ensures easy access to order details even for users without an account. (Logged-in customers can also use this form.)

Reviewer: ```suggestion
- The `OrderSearch` container enables guest users to locate their orders using a combination of email, ZIP code, and order number. This ensures easy access to order details even for users without an account. (Logged-in customers can also use this form.)
```

> The drop-in's initialization process helps manage data retrieval and event emission, ensuring that containers receive the necessary data without individual fetching.

Reviewer: ```suggestion
The component's initialization process helps manage data retrieval and event emission, ensuring that containers receive the necessary data without individual fetching.
```
